packages/js-sdk/src/Dm3Sdk.test.ts

```diff
@@ -1,34 +1,278 @@
+import { StorageAPI } from '@dm3-org/dm3-lib-storage';
+import {
+    getMockDeliveryServiceProfile,
+    MockDeliveryServiceProfile,
+    MockedUserProfile,
+    MockMessageFactory,
+    mockUserProfile,
+} from '@dm3-org/dm3-lib-test-helper';
+import axios from 'axios';
 import { ethers } from 'ethers';
 import { Dm3Sdk, Dm3SdkConfig } from './Dm3Sdk';
-import { StorageAPI } from '@dm3-org/dm3-lib-storage';
+
+import MockAdapter from 'axios-mock-adapter';
+import { normalizeEnsName } from '@dm3-org/dm3-lib-profile';
+import { ITLDResolver } from './tld/nameService/ITLDResolver';
 
 describe('Dm3Sdk', () => {
-    let upController: ethers.Signer;
+    let alice: MockedUserProfile;
+    let bob: MockedUserProfile;
+    let karl: MockedUserProfile;
+
+    //Axios mock to mock the http requests
+    let axiosMock;
+
+    let deliveryService: MockDeliveryServiceProfile;
 
     beforeEach(async () => {
```

Comment:
would it make more sense to put the axios mocks in `beforeAll` ?

Reply:
The axios mocks have the address as a dependency, which is generated beforeEach the new test. i think its better to create a new wallet with beforeEach for every test to provide a clean environment